src/Models/ApplicantModel.php

```diff
@@ -521,4 +521,15 @@ public function updateApplicantStageAndOptionIds(int $applicantId, int $stageId,
         );
         return $query->execute([':applicantId' => $applicantId, ':stageId' => $stageId, ':optionId' => $optionId]);
     }
+
+    public function getAssessmentApplicants(): array
+    {
+        $stmt = "SELECT `applicants`.`id`, `applicants`.`name`, `applicants`.`email`, 
+                    `applicants_additional`.`assessmentDay` 
+                FROM `applicants` INNER JOIN `applicants_additional` 
+                ON `applicants`.`id`=`applicants_additional`.`id`;";
+        $query = $this->db->prepare($stmt);
+        $query->execute();
+        return $query->fetchAll();
+    }
```

Comment:
This query doesn't account for deleted applicants.

Reply:
Will speak to the people on s10 who wrote this!